content/python/concepts/strings/terms/casefold/casefold.md

```diff
@@ -13,7 +13,7 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Takes in a string, and returns a copy of the string with all characters in lower case.
+Takes in a string, and returns a copy of the string with all characters in lower case. `casefold()` is similar to `lower()`, but whereas that method deals purely with ASCII text, `casefold()` can convert Unicode characters. Take the German lowercase letter "ß", for example. `lower()` does not convert it, but `casefold()` converts it to "ss".
```

Comment:
```suggestion
Takes in a string, and returns a copy of the string with all characters in lower case. `.casefold()` is similar to `.lower()`, but whereas that method deals purely with ASCII text, `.casefold()` can also convert Unicode characters.
```

Reply:
Since you added a codebyte example with the German sharp-S, I moved some of the text from here to there. 

Also, when referring to functions/methods in Docs entries, let's place a period before the method name: `.method()`